packages/gatsby-theme-newrelic/src/components/ContributingGuidelines.js

```diff
@@ -101,8 +101,8 @@ const ContributingGuidelines = ({ fileRelativePath, pageTitle }) => {
             text-align: center;
           `}
         >
-          Read our <ExternalLink href={contributingUrl}>guide</ExternalLink> on
-          how to contribute
+          Suggest a change or learn how to
+          <ExternalLink href={contributingUrl}>contribute</ExternalLink>
```

Comment:
Ah looks like one place where we are currently using `ExternalLink`. I'd like to deprecate that component if we can in favor or `Link` from the theme. `Link` has all of the same functionality built, so we can maintain a single component. Could you update this to use `Link`?

Bonus: By using `Link`, this will handle both if the `contributingUrl` is either external or internal. If its an internal link, then it will ensure it transitions nicely to the url in the site rather than opening the page in a new link.

NOTE: You'll need to [update the import](https://github.com/newrelic/gatsby-theme-newrelic/pull/260/files#diff-fd108bf279d260e105ada43a84784852755997a1d2c377531ad94c3a779ab780R6) to pull from `'./Link'` instead of `'./ExternalLink'`

```suggestion
          <Link to={contributingUrl}>contribute</Link>
```